### ISSUE TITLE
Events on map

### ISSUE THREAD

**Issue by schradert** [Couchers-org/couchers#4514]

**Comment by bakeiro** [comment]
current situation:
No events on the map

desired situation
I want a pill that will show the events on the map

tasks:
Implement a pill system, which toggle which content you see on the map (events or people)

mockup:
![image](https://github.com/user-attachments/assets/50c57b40-d244-4b39-ab1b-129709baa7aa)

**Comment by nabramow** [comment]
@bakeiro I remember us talking about this not really being a priority anytime soon in a meeting. Should this move to the Backlog/Ideas? Or can a volunteer pick this up?

**Comment by bakeiro** [comment]
@nabramow I think this is a "should have" priority, IMHO I would sort it that way, we can speak about this in the meeting/slack :)

**Comment by nabramow** [comment]
@bakeiro Probably we want to do the map fixes before we start adding more to the map? But yeah let's talk about it. I'm gonna be late to the meeting but if you don't mind bringing it up?

**Comment by bakeiro** [comment]
Yeah, the map fixes are for me the "must have" then comes issues like this one :)
